otelib/pipe.py

```diff
@@ -13,6 +15,6 @@ class Pipe:
     def __init__(self, strategy: "AbstractStrategy") -> None:
         self.input: "AbstractStrategy" = strategy
 
-    def get(self, session: "Optional[str]" = None) -> bytes:
+    def get(self, session: "Optional[AttrDict]" = None) -> bytes:
```

Comment:
This is wrong, it's a `str`, specifically, the session ID.
```suggestion
    def get(self, session: "Optional[str]" = None) -> bytes:
```

Reply:
Then the argument is very misleading. Change it to `session_id`.

Reply:
The SDKdemo relies on these changes. If you want to reject this PR, then please update the SDKdemo such that it runs. My calendar is fully booked from now on to December 9th.

Reply:
Please open an issue for this outlining what changes are necessary and I'll take a look.

Reply:
> Then the argument is misleading. Change it to `session_id`.

Agreed.